docs/Getting Started/Fedora/Fedora Root On ZFS.rst

```diff
@@ -0,0 +1,751 @@
+           ... \
+           rpool mirror \
+           /dev/disk/by-id/scsi-SATA_disk1-part3 \
+           /dev/disk/by-id/scsi-SATA_disk2-part3
+
+   - For raidz topologies, replace ``mirror`` in the above command with
+     ``raidz``, ``raidz2``, or  ``raidz3`` and list the partitions from
+     additional disks.
+   - When using LUKS with mirror or raidz topologies, use
+     ``/dev/mapper/luks1``, ``/dev/mapper/luks2``, etc., which you will have
+     to create using ``cryptsetup``.
+   - The pool name is arbitrary. If changed, the new name must be used
+     consistently. On systems that can automatically install to ZFS, the root
+     pool is named ``rpool`` by default.
+
+Step 3: System Installation
+---------------------------
+
+#. Create the filesystem datasets to act as containers::
+
+     zfs create -o canmount=off -o mountpoint=none rpool/ROOT
+
+   On Solaris systems, the root filesystem is cloned and the suffix is
+   incremented for major system changes through ``pkg image-update`` or
+   ``beadm``. Similar functionality has not yet been implemented into Fedora and will most likely never be added to Fedora in the forseeable future due to licensing issues.
+
+#. Create filesystem datasets for the root and boot filesystems::
+
+     zfs create -o canmount=noauto -o mountpoint=/ rpool/ROOT/fedora
+     zfs mount rpool/ROOT/fedora
+
+   With ZFS, it is not normally necessary to use a mount command (either
+   ``mount`` or ``zfs mount``). This situation is an exception because of
+   ``canmount=noauto``.
+
+#. Create datasets::
+
+     zfs create                                 rpool/home
+     zfs create -o mountpoint=/root             rpool/home/root
+     zfs create -o canmount=off                 rpool/var
+     zfs create -o canmount=off                 rpool/var/lib
+     zfs create                                 rpool/var/log
+     zfs create                                 rpool/var/spool
+
+   The datasets below are optional, depending on your preferences and/or
+   software choices.
+
+   If you wish to exclude these from snapshots::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/cache
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/tmp
+     chmod 1777 /mnt/var/tmp
+
+   If this system will use Docker (which manages its own datasets &
+   snapshots)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/docker
+
+   If this system will use NFS (locking)::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/var/lib/nfs
+
+   A tmpfs is recommended later, but if you want a separate dataset for
+   ``/tmp``::
+
+     zfs create -o com.sun:auto-snapshot=false  rpool/tmp
+     chmod 1777 /mnt/tmp
+   
+   Note that the reason why we are not fully seperating everything like we did in Ubuntu is because dnf will fail to install or update certain packages if we create too many datasets. An example of one such package is filesystem, which fails to install if other ZFS datasets are created.
```

Comment:
```suggestion
   Note that the reason why we are not fully separating everything like we did in Ubuntu is because dnf will fail to install or update certain packages if we create too many datasets. An example of one such package is filesystem, which fails to install if other ZFS datasets are created.
```